clients/linux/src/backend.rs

```diff
@@ -168,6 +168,10 @@ impl LbCore {
         Ok(get_children(&self.config, parent.id)?)
     }
 
+    pub fn get_all_files(&self) -> LbResult<Vec<DecryptedFileMetadata>> {
+        Ok(list_metadatas(&self.config)?)
```

Comment:
This is not the way you want to do this, there already is a get all metas just think about how android is doing it!

Reply:
whoops I thought this was lib.rs for a second, nvm